Source/WebCore/testing/Internals.cpp

```diff
@@ -1022,7 +1022,8 @@ void Internals::setOverrideResourceLoadPriority(ResourceLoadPriority priority)
 
 void Internals::setStrictRawResourceValidationPolicyDisabled(bool disabled)
 {
-    frame()->loader().setStrictRawResourceValidationPolicyDisabledForTesting(disabled);
+    if (auto* localFrame = frame())
```

Comment:
I’d think this would need to be `RefPtr` instead of `auto*` in our new safer C++ model.

Reply:
For this case (and some others below) I think `loader()` and `setStrictRawResourceValidationPolicyDisabledForTesting()` are just trivial functions so that should be fine. But I agree let's try to use more smart pointers here, especially since the extra overhead is probably not important for these `internals` functions.

Reply:
Done.